skype/skype-ps/skype/Grant-CsTeamsUpgradePolicy.md

```diff
@@ -28,6 +28,8 @@ TeamsUpgradePolicy allows administrators to manage the transition from Skype for
 
 This cmdlet enables admins to apply TeamsUpgradePolicy to either individual users or to set the default for the entire organization. 
 
+**[NOTE]** Gallatin Tenants must run Start-CsExMeetingMigration after executing Grant-CsTeamsUpgradePolicy. Please see [Start-CsExMeetingMigrationService](https://learn.microsoft.com/en-us/powershell/module/skype/start-csexmeetingmigration?view=skype-ps).
```

Comment:
Earlier versions of this cmdlet used to support -MigrateMeetingsToTeams option. This option has been deprecated in newer versions of the module. Tenants must run Start-CsExMeetingMigration ....